files/methods.py

```diff
@@ -347,6 +349,54 @@ def notify_user_on_comment(friendly_token):
     return True
 
 
+def notify_user_on_mention(friendly_token, user_mentioned, cleaned_comment):
+    media = None
```

Comment:
I am refering to the `media = None` here. 

media on this queryset will be None if it doesn't find one, that's why I commented that `media = None` is redundant!
```
media = models.Media.objects.filter(friendly_token=friendly_token).first()
```